daemon/api_themes.go

```diff
@@ -227,16 +232,26 @@ func installThemes(c *Command, r *http.Request, user *auth.UserState) Response {
 	if user != nil {
 		userID = user.ID
 	}
-	installed, tasksets, err := snapstateInstallMany(st, toInstall, nil, userID, &snapstate.Flags{})
+
+	// TODO: should this use the request context?
```

Comment:
I don't understand this TODO? isn't it what the code is doing?

Reply:
It was more of a question if it is the right thing to do. Previously we didn't support a context for InstallMany operations.